src/main/java/com/synopsys/integration/detect/workflow/blackduck/developer/blackduck/DetectRapidScanService.java

```diff
@@ -54,26 +48,18 @@ public static DetectRapidScanService fromBlackDuckServicesFactory(BlackDuckServi
             new Bdio2ContentExtractor(),
             bdio2Uploader
         );
-        return new DetectRapidScanService(bdio2FileUploadService, rapidScanWaiter);
+        return new DetectRapidScanService(bdio2FileUploadService);
     }
 
-    public RapidScanResult performUpload(UploadBatch uploadBatch, long timeoutInSeconds, int waitIntervalInSeconds) throws IntegrationException, InterruptedException {
-        List<DeveloperScanComponentResultView> allScanComponents = new LinkedList<>();
+    public List<HttpUrl> performUpload(UploadBatch uploadBatch) throws IntegrationException {
         List<HttpUrl> allScanUrls = new LinkedList<>();
 
         for (UploadTarget uploadTarget : uploadBatch.getUploadTargets()) {
-            RapidScanResult scanResults = performUpload(uploadTarget, timeoutInSeconds, waitIntervalInSeconds);
-            allScanComponents.addAll(scanResults.getComponentResultViews());
-            allScanUrls.addAll(scanResults.getScanIds());
+            HttpUrl url = bdio2FileUploadService.uploadFile(uploadTarget);
+            logger.info("Starting Rapid Scan: {}", url);
```

Comment:
Can we also change the verbiage: "Completed Rapid Scan Upload"

Reply:
I think that would be confusing because of the word completed. I could easily see someone reading the log being confused. I could say "Rapid Scan Uploaded: "

Reply:
Or how about "Uploaded Rapid Scan: "?

Reply:
I like `Uploaded Rapid Scan: `